### PR TITLE
Remove libsbml refs

### PR DESCRIPTION
**Pull request by skeating**
## Description

changes many remaining libsbml references
and got it building on Windows

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated all documentation necessary.
- [ ] I have checked spelling in (new) comments.

## Testing
- [ ] Testing is done automatically and codecov shows test coverage
- [ ] This cannot be tested automatically

### CODE REVIEW

**Comment by skeating** [comment]
Hi Matthew

If you could check that this still builds for you.

I solved by dllimport issue and I think I told cmake about bindings so it may help with that too.

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ###############################################################################
 #
-# Description       : CMake build script for libSBML
+# Description       : CMake build script for libLX
```
Comment (mattagape) [inline CMakeLists.txt:3]: I believe the following lines in this CMakeLists.txt need to be updated: 139, 1482, 1495, 1505 (I'm not sure about the last two)

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -102,22 +91,76 @@ configure_file(
...
-file(GLOB header_files "${CMAKE_CURRENT_SOURCE_DIR}/liblx/xml/*.h")
-install(FILES ${header_files} DESTINATION include/liblx/xml)
+foreach(dir common compress xml)
+	file(GLOB header_files "${CMAKE_CURRENT_SOURCE_DIR}/liblx/xml/${dir}/*.h")
+	install(FILES ${header_files} DESTINATION include/liblx/xml/${dir})
+
+endforeach(dir)
+
+
+file(GLOB files "${CMAKE_CURRENT_SOURCE_DIR}/sbml/*.h")
```
Comment (mattagape) [inline src/CMakeLists.txt:101]: Mentions sbml.

File: src/CMakeLists.txt [hunk 3]
```diff
@@ -102,22 +91,76 @@ configure_file(
...
+install(FILES ${files} DESTINATION include/sbml)
```
Comment (mattagape) [inline src/CMakeLists.txt:102]: also mentions sbml

File: src/CMakeLists.txt [hunk 4]
```diff
@@ -365,46 +408,67 @@ endif()
...
 #
 add_subdirectory(liblx)
+##############################################################################
+#
+# construct list of all header files to create dependency list for 
+# language bindings
+#
+
+file(GLOB LIBLX_HEADER_FILES "${CMAKE_CURRENT_BINARY_DIR}/liblx/common/*.h")    
+foreach(file ${LIBSBML_SOURCES})
```
Comment (mattagape) [inline src/CMakeLists.txt:461]: Please check if we still need this. Also, line 231, do we need it?

File: src/bindings/swig/OStream.h [hunk 5]
```diff
@@ -90,7 +90,7 @@
 #include <fstream>
 #include <sstream>
 
-#include <liblx/xml/XMLExtern.h>
+#include <liblx/xml/common/extern.h>
```
Comment (mattagape) [inline src/bindings/swig/OStream.h:93]: Some or all of the following lines need changing, I believe:
7,8,34,42,43,60,62,77,79

File: src/liblx/xml/LibXMLTranscode.h [hunk 6]
```diff
@@ -43,7 +43,7 @@
 
 #include <string>
 #include <libxml/parser.h>
-#include <liblx/xml/XMLExtern.h>
+#include <liblx/xml/common/extern.h>
```
Comment (mattagape) [inline src/liblx/xml/LibXMLTranscode.h:47]: query lines 2,9,10,36

**Review by mattagape — APPROVED** [review]
There are a few instances of "sbml" terms still floating around, but I think we can mop those up later.

**Pull request merged**